src/ec/packages/ec-component-gallery/gallery.story.js

```diff
@@ -1,29 +1,77 @@
+/* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
-import { withKnobs } from '@storybook/addon-knobs';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import { getExtraKnobs, buttonLabels } from '@ecl-twig/story-utils';
 import withCode from '@ecl-twig/storybook-addon-code';
 
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import gallery from './ecl-gallery.html.twig';
 
-import data from './demo/data';
+import dataDefault from './demo/data';
 import notes from './README.md';
 
-data.items.forEach(item => {
-  if (item.icon) {
-    item.icon.path = defaultSprite; // eslint-disable-line no-param-reassign
-  }
-});
-data.overlay.close.icon.path = defaultSprite;
-data.overlay.previous.icon.path = defaultSprite;
-data.overlay.next.icon.path = defaultSprite;
-data.overlay.download.icon.path = defaultSprite;
-data.overlay.share.icon.path = defaultSprite;
+const preperaGallery = data => {
+  data.items.forEach((item, index) => {
+    item.share_path = select(
```

Comment:
What about the other parameters?
- "path"
- "alt"
- "meta"
 - "icon"

Reply:
i didn't see them in the documentation in ecl for the gallery but i will put them in